docusaurus/docs/extensions/api/concepts.md

```diff
@@ -1 +1,66 @@
 # Concepts
+
+Extensions largely work the same as the rest of the dashboard. There are a set of top-level folders that can be defined and used as they are in the dashboard: `chart`, `cloud-credential`, `content`, `detail`, `edit`, `list`, `machine-config`, `models`, `promptRemove`, `l10n`, `windowComponents`, `dialog`, and `formatters`. You can read about what each of these folders does [here](../../code-base-works/directory-structure.md).
```

Comment:
Can we remove this paragraph - some bits covered elsewhere and will add back in the rest overtime

Reply:
removed 🙏